src/ansys/aedt/core/modeler/cad/primitives.py

```diff
@@ -1481,6 +1481,58 @@ def cover_faces(self, assignment):
         self.oeditor.CoverSurfaces(["NAME:Selections", "Selections:=", obj_to_cover, "NewPartsModelFlag:=", "Model"])
         return True
 
+    @pyaedt_function_handler(selection="assignment")
+    def uncover_faces(self, assignment):
+        """Uncover surfaces/faces.
```

Comment:
```suggestion
        """Uncover faces.
```
Should not be only faces? In AEDT I do not think surfaces has a meaning